2019-10-09 16:02:36.420  INFO [shineon.server.uauth,,,] 8136 --- [main] com.shineon.coder.CoderApplication       : No active profile set, falling back to default profiles: default
2019-10-09 16:02:37.025  INFO [shineon.server.uauth,,,] 8136 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2019-10-09 16:02:37.026  INFO [shineon.server.uauth,,,] 8136 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-10-09 16:02:37.042  INFO [shineon.server.uauth,,,] 8136 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 9ms. Found 0 repository interfaces.
2019-10-09 16:02:37.106  WARN [shineon.server.uauth,,,] 8136 --- [main] o.s.boot.actuate.endpoint.EndpointId     : Endpoint ID 'service-registry' contains invalid characters, please migrate to a valid format.
2019-10-09 16:02:37.349  INFO [shineon.server.uauth,,,] 8136 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=a08bfaf0-03dd-3ed9-8dc1-d39615d2f88a
2019-10-09 16:02:37.358  INFO [shineon.server.uauth,,,] 8136 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'errorChannel' has been explicitly defined. Therefore, a default PublishSubscribeChannel will be created.
2019-10-09 16:02:37.362  INFO [shineon.server.uauth,,,] 8136 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'taskScheduler' has been explicitly defined. Therefore, a default ThreadPoolTaskScheduler will be created.
2019-10-09 16:02:37.366  INFO [shineon.server.uauth,,,] 8136 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'integrationHeaderChannelRegistry' has been explicitly defined. Therefore, a default DefaultHeaderChannelRegistry will be created.
2019-10-09 16:02:37.581  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'integrationDisposableAutoCreatedBeans' of type [org.springframework.integration.config.annotation.Disposables] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.610  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.integration.config.IntegrationManagementConfiguration' of type [org.springframework.integration.config.IntegrationManagementConfiguration$$EnhancerBySpringCGLIB$$7e934ef] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.621  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration' of type [org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration$$EnhancerBySpringCGLIB$$e37b400f] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.629  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration' of type [org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration$$EnhancerBySpringCGLIB$$1c62e8dc] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.634  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'mbeanServer' of type [com.sun.jmx.mbeanserver.JmxMBeanServer] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.664  INFO [shineon.server.uauth,,,] 8136 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$b9117cbd] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:02:37.999  INFO [shineon.server.uauth,,,] 8136 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 122 (http)
2019-10-09 16:02:38.018  INFO [shineon.server.uauth,,,] 8136 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-10-09 16:02:38.018  INFO [shineon.server.uauth,,,] 8136 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.21]
2019-10-09 16:02:38.126  INFO [shineon.server.uauth,,,] 8136 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-10-09 16:02:38.127  INFO [shineon.server.uauth,,,] 8136 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1683 ms
2019-10-09 16:02:38.245  WARN [shineon.server.uauth,,,] 8136 --- [main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2019-10-09 16:02:38.245  INFO [shineon.server.uauth,,,] 8136 --- [main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2019-10-09 16:02:38.251  INFO [shineon.server.uauth,,,] 8136 --- [main] c.netflix.config.DynamicPropertyFactory  : DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@34a20f16
2019-10-09 16:02:39.275  INFO [shineon.server.uauth,,,] 8136 --- [main] o.s.s.c.ThreadPoolTaskScheduler          : Initializing ExecutorService 'taskScheduler'
2019-10-09 16:02:39.934  WARN [shineon.server.uauth,,,] 8136 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userController': Unsatisfied dependency expressed through field 'userDTO'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userDTO': Unsatisfied dependency expressed through field 'base'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'com.shineon.coder.service.feign.UserBase': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'userBaseFallback': Unsatisfied dependency expressed through field 'itemUtils'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.shineon.coder.service.convert.CommonItemUtils' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
2019-10-09 16:02:39.937  INFO [shineon.server.uauth,,,] 8136 --- [main] o.s.s.c.ThreadPoolTaskScheduler          : Shutting down ExecutorService 'taskScheduler'
2019-10-09 16:02:39.949  INFO [shineon.server.uauth,,,] 8136 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2019-10-09 16:02:39.965  INFO [shineon.server.uauth,,,] 8136 --- [main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2019-10-09 16:02:40.128 ERROR [shineon.server.uauth,,,] 8136 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Field itemUtils in com.shineon.coder.service.feign.UserBaseFallback required a bean of type 'com.shineon.coder.service.convert.CommonItemUtils' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)


Action:

Consider defining a bean of type 'com.shineon.coder.service.convert.CommonItemUtils' in your configuration.

2019-10-09 16:08:01.694  INFO [shineon.server.uauth,,,] 13576 --- [main] com.shineon.coder.CoderApplication       : No active profile set, falling back to default profiles: default
2019-10-09 16:08:02.276  INFO [shineon.server.uauth,,,] 13576 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2019-10-09 16:08:02.277  INFO [shineon.server.uauth,,,] 13576 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-10-09 16:08:02.293  INFO [shineon.server.uauth,,,] 13576 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 9ms. Found 0 repository interfaces.
2019-10-09 16:08:02.356  WARN [shineon.server.uauth,,,] 13576 --- [main] o.s.boot.actuate.endpoint.EndpointId     : Endpoint ID 'service-registry' contains invalid characters, please migrate to a valid format.
2019-10-09 16:08:02.604  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=a08bfaf0-03dd-3ed9-8dc1-d39615d2f88a
2019-10-09 16:08:02.613  INFO [shineon.server.uauth,,,] 13576 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'errorChannel' has been explicitly defined. Therefore, a default PublishSubscribeChannel will be created.
2019-10-09 16:08:02.618  INFO [shineon.server.uauth,,,] 13576 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'taskScheduler' has been explicitly defined. Therefore, a default ThreadPoolTaskScheduler will be created.
2019-10-09 16:08:02.621  INFO [shineon.server.uauth,,,] 13576 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'integrationHeaderChannelRegistry' has been explicitly defined. Therefore, a default DefaultHeaderChannelRegistry will be created.
2019-10-09 16:08:02.839  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'integrationDisposableAutoCreatedBeans' of type [org.springframework.integration.config.annotation.Disposables] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:02.873  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.integration.config.IntegrationManagementConfiguration' of type [org.springframework.integration.config.IntegrationManagementConfiguration$$EnhancerBySpringCGLIB$$a525c8d0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:02.888  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration' of type [org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration$$EnhancerBySpringCGLIB$$80b7d3f0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:02.896  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration' of type [org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration$$EnhancerBySpringCGLIB$$b99f7cbd] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:02.901  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'mbeanServer' of type [com.sun.jmx.mbeanserver.JmxMBeanServer] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:02.933  INFO [shineon.server.uauth,,,] 13576 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$564e109e] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:08:03.265  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 122 (http)
2019-10-09 16:08:03.280  INFO [shineon.server.uauth,,,] 13576 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-10-09 16:08:03.280  INFO [shineon.server.uauth,,,] 13576 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.21]
2019-10-09 16:08:03.383  INFO [shineon.server.uauth,,,] 13576 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-10-09 16:08:03.383  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1680 ms
2019-10-09 16:08:03.505  WARN [shineon.server.uauth,,,] 13576 --- [main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2019-10-09 16:08:03.505  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2019-10-09 16:08:03.511  INFO [shineon.server.uauth,,,] 13576 --- [main] c.netflix.config.DynamicPropertyFactory  : DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@14de1901
2019-10-09 16:08:04.507  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.s.c.ThreadPoolTaskScheduler          : Initializing ExecutorService 'taskScheduler'
2019-10-09 16:08:05.197  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2019-10-09 16:08:05.197  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'pool_schedule'
2019-10-09 16:08:05.271  WARN [shineon.server.uauth,,,] 13576 --- [main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2019-10-09 16:08:05.271  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2019-10-09 16:08:06.143  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2019-10-09 16:08:06.225  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.s.m.DirectWithAttributesChannel    : Channel 'application-1.Matao2MessageInput' has 1 subscriber(s).
2019-10-09 16:08:06.318  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel nullChannel
2019-10-09 16:08:06.329  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel errorChannel
2019-10-09 16:08:06.370  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageInput
2019-10-09 16:08:06.390  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageOutput
2019-10-09 16:08:06.400  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageHandler errorLogger
2019-10-09 16:08:06.412  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageHandler org.springframework.cloud.stream.binding.StreamListenerMessageHandler@788d9139
2019-10-09 16:08:06.464  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.endpoint.EventDrivenConsumer       : Adding {logging-channel-adapter:_org.springframework.integration.errorLogger} as a subscriber to the 'errorChannel' channel
2019-10-09 16:08:06.464  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.channel.PublishSubscribeChannel    : Channel 'application-1.errorChannel' has 1 subscriber(s).
2019-10-09 16:08:06.464  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.endpoint.EventDrivenConsumer       : started _org.springframework.integration.errorLogger
2019-10-09 16:08:06.642  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:06.671 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:06.676  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.s.m.DirectWithAttributesChannel    : Channel 'application-1.Matao2MessageOutput' has 1 subscriber(s).
2019-10-09 16:08:06.688  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.n.eureka.InstanceInfoFactory       : Setting initial instance status as: STARTING
2019-10-09 16:08:06.713  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Initializing Eureka in region us-east-1
2019-10-09 16:08:06.801  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2019-10-09 16:08:06.801  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2019-10-09 16:08:07.017  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2019-10-09 16:08:07.017  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2019-10-09 16:08:07.148  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:08:07.188  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2019-10-09 16:08:07.188  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2019-10-09 16:08:07.189  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2019-10-09 16:08:07.189  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Application is null : false
2019-10-09 16:08:07.189  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2019-10-09 16:08:07.189  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Application version is -1: true
2019-10-09 16:08:07.189  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2019-10-09 16:08:07.284  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : The response status is 200
2019-10-09 16:08:07.286  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Starting heartbeat executor: renew interval is: 30
2019-10-09 16:08:07.289  INFO [shineon.server.uauth,,,] 13576 --- [main] c.n.discovery.InstanceInfoReplicator     : InstanceInfoReplicator onDemand update allowed rate per min is 4
2019-10-09 16:08:07.292  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1570608487291 with initial instances count: 2
2019-10-09 16:08:07.297  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.n.e.s.EurekaServiceRegistry        : Registering application SHINEON.SERVER.UAUTH with eureka with status UP
2019-10-09 16:08:07.297  INFO [shineon.server.uauth,,,] 13576 --- [main] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1570608487297, current=UP, previous=STARTING]
2019-10-09 16:08:07.298  INFO [shineon.server.uauth,,,] 13576 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SHINEON.SERVER.UAUTH/fh18829-2:shineon.server.uauth:122: registering service...
2019-10-09 16:08:07.316  INFO [shineon.server.uauth,,,] 13576 --- [main] c.s.b.r.p.RabbitExchangeQueueProvisioner : declaring queue for inbound: Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w, bound to: Matao2MessageInput
2019-10-09 16:08:07.317  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:07.324  INFO [shineon.server.uauth,,,] 13576 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SHINEON.SERVER.UAUTH/fh18829-2:shineon.server.uauth:122 - registration status: 204
2019-10-09 16:08:07.326  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:07.326 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:07.330 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:07.332  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:07.336 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:07.345  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w.errors
2019-10-09 16:08:07.396  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.stream.binder.BinderErrorChannel   : Channel 'application-1.Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w.errors' has 1 subscriber(s).
2019-10-09 16:08:07.397  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.c.stream.binder.BinderErrorChannel   : Channel 'application-1.Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w.errors' has 2 subscriber(s).
2019-10-09 16:08:07.404  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:07.408  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.a.r.l.SimpleMessageListenerContainer : Broker not available; cannot force queue declarations during start: java.io.IOException
2019-10-09 16:08:07.409 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:07.413  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-1] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:07.417 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:07.419  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.i.a.i.AmqpInboundChannelAdapter      : started inbound.Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w
2019-10-09 16:08:07.465  INFO [shineon.server.uauth,,,] 13576 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 122 (http) with context path ''
2019-10-09 16:08:07.466  INFO [shineon.server.uauth,,,] 13576 --- [main] .s.c.n.e.s.EurekaAutoServiceRegistration : Updating port to 122
2019-10-09 16:08:07.467  INFO [shineon.server.uauth,,,] 13576 --- [main] com.shineon.coder.CoderApplication       : Started CoderApplication in 6.536 seconds (JVM running for 7.036)
2019-10-09 16:08:12.421  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-1] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3733b1f4: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:12.422  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-2] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:12.427 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:17.427  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-2] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@166ef587: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:17.428  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-3] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:17.432 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:22.432  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-3] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@da8c59a: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:22.432  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-4] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:22.436 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:27.436  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-4] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@45242dc6: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:27.436  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-5] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:27.442 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:32.442  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-5] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@532844e8: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:32.442  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-6] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:32.447 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:37.446  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-6] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3cdaba70: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:37.446  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-7] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:37.451 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:42.450  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-7] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@78e5da9f: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:42.450  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-8] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:42.455 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:47.454  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-8] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@abae14: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:47.454  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-9] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:47.460 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:52.460  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-9] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@45c2d4ed: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:52.460  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-10] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:52.465 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:08:57.465  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-10] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@219ed705: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:08:57.465  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-11] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:08:57.470 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:02.470  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-11] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3693773b: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:02.470  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-12] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:02.474 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:07.474  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-12] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@120ba60f: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:07.474  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-13] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:07.479 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:12.479  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-13] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@73a16e03: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:12.479  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-14] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:12.484 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:17.483  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-14] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@596cbf49: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:17.483  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-15] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:17.488 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:22.487  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-15] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@29fe2ee: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:22.487  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-16] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:22.492 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:27.492  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-16] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3075fe04: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:27.492  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-17] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:27.497 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:32.498  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-17] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5f36913b: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:32.498  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-18] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:32.502 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:37.502  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-18] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@1307e5f5: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:37.502  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-19] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:37.507 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:42.507  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-19] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@4a06d6d0: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:42.507  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-20] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:42.512 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:47.512  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-20] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@29bedf88: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:47.513  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-21] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:47.517 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:52.517  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-21] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@61972b83: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:52.517  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-22] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:52.528 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: Socket Closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:09:57.528  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-22] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@2a62f8da: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:09:57.528  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-23] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:09:57.533 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:02.533  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-23] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@1ae7d962: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:02.533  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-24] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:02.538 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:07.538  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-24] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3ccb1f94: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:07.538  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-25] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:07.543 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:12.543  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-25] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5e0eb408: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:12.543  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-26] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:12.547 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:17.547  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-26] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6f827514: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:17.547  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-27] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:17.550 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:22.550  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-27] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@2a6fdd55: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:22.550  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-28] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:22.554 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:27.554  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-28] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@40f2a0a2: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:27.554  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-29] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:27.559 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:32.559  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-29] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5ed515f8: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:32.559  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-30] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:32.563 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:37.563  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-30] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@7b3269ec: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:37.563  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-31] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:37.567 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:42.567  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-31] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6655315c: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:42.567  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-32] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:42.572 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:47.572  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-32] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6f315aa3: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:47.572  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-33] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:47.577 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:52.576  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-33] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@48cfbd74: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:52.576  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-34] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:52.581 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:10:57.581  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-34] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@550f0b86: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:10:57.581  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-35] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:10:57.586 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:02.585  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-35] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@ba553e5: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:02.585  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-36] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:02.589 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:07.589  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-36] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@162f13e: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:07.589  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-37] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:07.594 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:12.593  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-37] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@16a84c9b: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:12.593  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-38] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:12.599 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:17.599  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-38] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5ed8e2d8: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:17.599  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-39] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:17.603 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:22.603  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-39] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@7d7761c1: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:22.603  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-40] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:22.608 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:27.607  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-40] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@7cf9983e: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:27.607  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-41] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:27.611 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:32.611  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-41] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@27fbd5bc: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:32.611  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-42] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:32.615 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:37.615  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-42] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@68e83219: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:37.615  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-43] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:37.619 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:42.619  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-43] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@53e26396: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:42.619  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-44] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:42.623 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:47.623  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-44] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@67af857a: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:47.623  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-45] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:47.627 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:52.627  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-45] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@333a433: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:52.627  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-46] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:52.632 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:11:57.632  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-46] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6a5de6d3: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:11:57.632  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-47] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:11:57.638 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:02.638  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-47] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5edbaee9: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:02.638  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-48] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:02.643 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:07.642  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-48] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5b6af7e6: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:07.642  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-49] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:07.647 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:12.647  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-49] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@2a08ea: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:12.647  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-50] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:12.651 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:17.651  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-50] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@1ce18433: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:17.651  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-51] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:17.655 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:22.655  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-51] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@414b2ba4: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:22.655  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-52] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:22.664 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:27.664  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-52] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@46676495: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:27.664  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-53] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:27.668 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:32.668  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-53] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@7cdc5ea3: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:32.668  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-54] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:32.672 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:37.672  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-54] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3d689523: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:37.672  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-55] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:37.676 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:42.676  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-55] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@58b9f187: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:42.676  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-56] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:42.681 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:47.681  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-56] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@4da456e: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:47.681  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-57] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:47.687 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:52.687  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-57] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@316838d6: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:52.687  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-58] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:52.691 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:12:57.690  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-58] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@7c3ba929: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:12:57.690  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-59] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:12:57.695 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:02.695  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-59] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@4e8aeab0: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:02.695  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-60] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:02.700 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:07.214  INFO [shineon.server.uauth,,,] 13576 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:13:07.700  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-60] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@709082e5: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:07.700  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-61] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:07.705 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:12.705  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-61] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@356a25ae: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:12.705  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-62] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:12.709 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:17.709  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-62] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@4c10cd9d: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:17.709  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-63] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:17.714 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:22.713  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-63] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@76577b1a: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:22.713  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-64] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:22.717 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: Socket Closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:27.717  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-64] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@2f557b92: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:27.718  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-65] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:27.722 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:32.722  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-65] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@252354f4: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:32.722  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-66] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:32.726 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:37.726  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-66] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@3da65fae: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:37.726  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-67] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:37.730 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:42.730  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-67] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@2fdc1b37: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:42.730  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-68] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:42.734 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: Socket Closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:47.734  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-68] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@718be792: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:47.734  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-69] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:47.738 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:52.738  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-69] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@5ac63ef2: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:52.738  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-70] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:52.742 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:13:57.742  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-70] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6400347f: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:13:57.742  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-71] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:13:57.746 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:02.750  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-71] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@1d656dae: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:02.750  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-72] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:02.754 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:07.754  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-72] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@6ec6bac7: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:07.754  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-73] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:07.758 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:12.758  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-73] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@78d0668e: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:12.758  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-74] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:12.762 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:17.762  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-74] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@15da662c: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:17.762  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-75] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:17.767 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:22.767  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-75] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@4231b79f: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:22.767  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-76] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:22.771 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:27.771  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-76] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@1cb35e83: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:27.771  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-77] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:27.775 ERROR [shineon.server.uauth,,,] 13576 --- [AMQP Connection 127.0.0.1:5672] c.r.c.impl.ForgivingExceptionHandler     : An unexpected connection driver error occured

java.net.SocketException: socket closed
	at java.net.SocketInputStream.socketRead0(Native Method) ~[na:1.8.0_60]
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:170) ~[na:1.8.0_60]
	at java.net.SocketInputStream.read(SocketInputStream.java:141) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246) ~[na:1.8.0_60]
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265) ~[na:1.8.0_60]
	at java.io.DataInputStream.readUnsignedByte(DataInputStream.java:288) ~[na:1.8.0_60]
	at com.rabbitmq.client.impl.Frame.readFrom(Frame.java:91) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.SocketFrameHandler.readFrame(SocketFrameHandler.java:164) ~[amqp-client-5.4.3.jar:5.4.3]
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:596) ~[amqp-client-5.4.3.jar:5.4.3]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_60]

2019-10-09 16:14:32.775  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-77] o.s.a.r.l.SimpleMessageListenerContainer : Restarting Consumer@498f9fd1: tags=[[]], channel=null, acknowledgeMode=AUTO local queue size=0
2019-10-09 16:14:32.775  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-78] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:32.783  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-78] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#734ce281:82/SimpleConnection@2cf11540 [delegate=amqp://matao@127.0.0.1:5672/, localPort= 54653]
2019-10-09 16:14:32.790  INFO [shineon.server.uauth,,,] 13576 --- [Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w-78] o.s.amqp.rabbit.core.RabbitAdmin         : Auto-declaring a non-durable, auto-delete, or exclusive Queue (Matao2MessageInput.anonymous.HGNIy8j2SSC6lbRlpS4J2w) durable:false, auto-delete:true, exclusive:true. It will be redeclared if the broker stops and is restarted while the connection factory is alive, but all messages will be lost.
2019-10-09 16:14:45.450  INFO [shineon.server.uauth,,,] 14560 --- [main] com.shineon.coder.CoderApplication       : No active profile set, falling back to default profiles: default
2019-10-09 16:14:46.025  INFO [shineon.server.uauth,,,] 14560 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2019-10-09 16:14:46.026  INFO [shineon.server.uauth,,,] 14560 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
2019-10-09 16:14:46.042  INFO [shineon.server.uauth,,,] 14560 --- [main] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 9ms. Found 0 repository interfaces.
2019-10-09 16:14:46.110  WARN [shineon.server.uauth,,,] 14560 --- [main] o.s.boot.actuate.endpoint.EndpointId     : Endpoint ID 'service-registry' contains invalid characters, please migrate to a valid format.
2019-10-09 16:14:46.345  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=a08bfaf0-03dd-3ed9-8dc1-d39615d2f88a
2019-10-09 16:14:46.356  INFO [shineon.server.uauth,,,] 14560 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'errorChannel' has been explicitly defined. Therefore, a default PublishSubscribeChannel will be created.
2019-10-09 16:14:46.360  INFO [shineon.server.uauth,,,] 14560 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'taskScheduler' has been explicitly defined. Therefore, a default ThreadPoolTaskScheduler will be created.
2019-10-09 16:14:46.364  INFO [shineon.server.uauth,,,] 14560 --- [main] faultConfiguringBeanFactoryPostProcessor : No bean named 'integrationHeaderChannelRegistry' has been explicitly defined. Therefore, a default DefaultHeaderChannelRegistry will be created.
2019-10-09 16:14:46.579  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'integrationDisposableAutoCreatedBeans' of type [org.springframework.integration.config.annotation.Disposables] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:46.609  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.integration.config.IntegrationManagementConfiguration' of type [org.springframework.integration.config.IntegrationManagementConfiguration$$EnhancerBySpringCGLIB$$1af6af7b] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:46.622  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration' of type [org.springframework.boot.autoconfigure.integration.IntegrationAutoConfiguration$IntegrationJmxConfiguration$$EnhancerBySpringCGLIB$$f688ba9b] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:46.629  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration' of type [org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration$$EnhancerBySpringCGLIB$$2f706368] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:46.634  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'mbeanServer' of type [com.sun.jmx.mbeanserver.JmxMBeanServer] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:46.664  INFO [shineon.server.uauth,,,] 14560 --- [main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$cc1ef749] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2019-10-09 16:14:47.032  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 122 (http)
2019-10-09 16:14:47.050  INFO [shineon.server.uauth,,,] 14560 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-10-09 16:14:47.050  INFO [shineon.server.uauth,,,] 14560 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.21]
2019-10-09 16:14:47.161  INFO [shineon.server.uauth,,,] 14560 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-10-09 16:14:47.161  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1702 ms
2019-10-09 16:14:47.277  WARN [shineon.server.uauth,,,] 14560 --- [main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2019-10-09 16:14:47.277  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2019-10-09 16:14:47.282  INFO [shineon.server.uauth,,,] 14560 --- [main] c.netflix.config.DynamicPropertyFactory  : DynamicPropertyFactory is initialized with configuration sources: com.netflix.config.ConcurrentCompositeConfiguration@681061d6
2019-10-09 16:14:48.293  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.s.c.ThreadPoolTaskScheduler          : Initializing ExecutorService 'taskScheduler'
2019-10-09 16:14:48.972  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2019-10-09 16:14:48.973  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'pool_schedule'
2019-10-09 16:14:49.047  WARN [shineon.server.uauth,,,] 14560 --- [main] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2019-10-09 16:14:49.047  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2019-10-09 16:14:50.024  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2019-10-09 16:14:50.131  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.s.m.DirectWithAttributesChannel    : Channel 'application-1.Matao2MessageInput' has 1 subscriber(s).
2019-10-09 16:14:50.224  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageOutput
2019-10-09 16:14:50.274  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel nullChannel
2019-10-09 16:14:50.284  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageInput
2019-10-09 16:14:50.289  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel errorChannel
2019-10-09 16:14:50.316  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageHandler errorLogger
2019-10-09 16:14:50.329  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageHandler org.springframework.cloud.stream.binding.StreamListenerMessageHandler@48bab2c6
2019-10-09 16:14:50.390  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.endpoint.EventDrivenConsumer       : Adding {logging-channel-adapter:_org.springframework.integration.errorLogger} as a subscriber to the 'errorChannel' channel
2019-10-09 16:14:50.390  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.channel.PublishSubscribeChannel    : Channel 'application-1.errorChannel' has 1 subscriber(s).
2019-10-09 16:14:50.390  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.endpoint.EventDrivenConsumer       : started _org.springframework.integration.errorLogger
2019-10-09 16:14:50.612  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.a.r.c.CachingConnectionFactory       : Attempting to connect to: [127.0.0.1:5672]
2019-10-09 16:14:50.634  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.a.r.c.CachingConnectionFactory       : Created new connection: rabbitConnectionFactory#11826398:0/SimpleConnection@70f76276 [delegate=amqp://matao@127.0.0.1:5672/, localPort= 54737]
2019-10-09 16:14:50.661  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.s.m.DirectWithAttributesChannel    : Channel 'application-1.Matao2MessageOutput' has 1 subscriber(s).
2019-10-09 16:14:50.672  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.n.eureka.InstanceInfoFactory       : Setting initial instance status as: STARTING
2019-10-09 16:14:50.695  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Initializing Eureka in region us-east-1
2019-10-09 16:14:50.777  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON encoding codec LegacyJacksonJson
2019-10-09 16:14:50.777  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using JSON decoding codec LegacyJacksonJson
2019-10-09 16:14:51.003  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML encoding codec XStreamXml
2019-10-09 16:14:51.003  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.d.provider.DiscoveryJerseyProvider   : Using XML decoding codec XStreamXml
2019-10-09 16:14:51.149  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Disable delta property : false
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Single vip registry refresh property : null
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Force full registry fetch : false
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Application is null : false
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Registered Applications size is zero : true
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Application version is -1: true
2019-10-09 16:14:51.188  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Getting all instance registry info from the eureka server
2019-10-09 16:14:51.295  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : The response status is 200
2019-10-09 16:14:51.297  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Starting heartbeat executor: renew interval is: 30
2019-10-09 16:14:51.299  INFO [shineon.server.uauth,,,] 14560 --- [main] c.n.discovery.InstanceInfoReplicator     : InstanceInfoReplicator onDemand update allowed rate per min is 4
2019-10-09 16:14:51.301  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Discovery Client initialized at timestamp 1570608891301 with initial instances count: 3
2019-10-09 16:14:51.306  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.n.e.s.EurekaServiceRegistry        : Registering application SHINEON.SERVER.UAUTH with eureka with status UP
2019-10-09 16:14:51.306  INFO [shineon.server.uauth,,,] 14560 --- [main] com.netflix.discovery.DiscoveryClient    : Saw local status change event StatusChangeEvent [timestamp=1570608891306, current=UP, previous=STARTING]
2019-10-09 16:14:51.307  INFO [shineon.server.uauth,,,] 14560 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SHINEON.SERVER.UAUTH/fh18829-2:shineon.server.uauth:122: registering service...
2019-10-09 16:14:51.327  INFO [shineon.server.uauth,,,] 14560 --- [main] c.s.b.r.p.RabbitExchangeQueueProvisioner : declaring queue for inbound: Matao2MessageInput.anonymous.6ZnrMkSPSTKlxuWE7xlYiQ, bound to: Matao2MessageInput
2019-10-09 16:14:51.334  INFO [shineon.server.uauth,,,] 14560 --- [DiscoveryClient-InstanceInfoReplicator-0] com.netflix.discovery.DiscoveryClient    : DiscoveryClient_SHINEON.SERVER.UAUTH/fh18829-2:shineon.server.uauth:122 - registration status: 204
2019-10-09 16:14:51.344  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.monitor.IntegrationMBeanExporter   : Registering MessageChannel Matao2MessageInput.anonymous.6ZnrMkSPSTKlxuWE7xlYiQ.errors
2019-10-09 16:14:51.396  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.stream.binder.BinderErrorChannel   : Channel 'application-1.Matao2MessageInput.anonymous.6ZnrMkSPSTKlxuWE7xlYiQ.errors' has 1 subscriber(s).
2019-10-09 16:14:51.397  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.c.stream.binder.BinderErrorChannel   : Channel 'application-1.Matao2MessageInput.anonymous.6ZnrMkSPSTKlxuWE7xlYiQ.errors' has 2 subscriber(s).
2019-10-09 16:14:51.415  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.i.a.i.AmqpInboundChannelAdapter      : started inbound.Matao2MessageInput.anonymous.6ZnrMkSPSTKlxuWE7xlYiQ
2019-10-09 16:14:51.460  INFO [shineon.server.uauth,,,] 14560 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 122 (http) with context path ''
2019-10-09 16:14:51.460  INFO [shineon.server.uauth,,,] 14560 --- [main] .s.c.n.e.s.EurekaAutoServiceRegistration : Updating port to 122
2019-10-09 16:14:51.461  INFO [shineon.server.uauth,,,] 14560 --- [main] com.shineon.coder.CoderApplication       : Started CoderApplication in 6.712 seconds (JVM running for 7.181)
2019-10-09 16:19:51.220  INFO [shineon.server.uauth,,,] 14560 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:24:51.244  INFO [shineon.server.uauth,,,] 14560 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:29:51.268  INFO [shineon.server.uauth,,,] 14560 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
2019-10-09 16:34:51.292  INFO [shineon.server.uauth,,,] 14560 --- [AsyncResolver-bootstrap-executor-0] c.n.d.s.r.aws.ConfigClusterResolver      : Resolving eureka endpoints via configuration
